code/datums/status_effects/neutral.dm

```diff
@@ -460,6 +460,17 @@
 		QDEL_NULL(alt_clone)
 	return ..()
 
+// A status effect that represents a character crawling
+/datum/status_effect/crawling
+	id = "crawling"
+	duration = 6 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = null
+
+/datum/status_effect/crawling/on_remove()
+	if(owner.body_position == LYING_DOWN)
+		owner.balloon_alert(owner, "You assume a prone position.")
```

Comment:
sorry that i gotta be a total stickler about this but balloon alert format does not include caps or punctuation, and should be a bit shorter

for future reference heres a handy dandy guide
normally: `balloon_alert(owner, "in prone position")`
failing something: `balloon_alert(owner, "interrupted!")`
something in progress (not really used much because progress bars are better): `balloon_alert(owner, "unwrenching...")`

Reply:
Thanks, sorry I'm a bit out of the loop.